CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
 option(USE_LLD "Use lld (from llvm) linker" OFF)
 option(USE_MOLD "Use mold (A Modern Linker)" OFF)
+option(TI_BUILD_CI "(Internal Use) Enable certain validations and possible debuggers for Taichi CI system" OFF)
```

Comment:
I somehow recall there is a TI_CI macro or something like that already. Can we reuse that name?

Reply:
Ah that's right, thanks!